### PR TITLE
17 add support for sqlite for wp now httpsplaygroundwordpressnet

### PR DESCRIPTION
**Pull request by remcotolsma**
```
npx wp-now start
```

### CODE REVIEW

File: adminer/index.php [hunk 1]
```diff
@@ -10,6 +10,16 @@ class PronamicAdminer extends Adminer {
...
 		}
+
+		/**
+		 * Login.
+		 *
+		 * @link https://www.adminer.org/en/password/
+		 * @link https://github.com/vrana/adminer/blob/7247f801bd06e51347d7ea671484e0fa6a883cbb/adminer/include/adminer.inc.php#L142-L152
+		 */
+		public function login( $login, $password ) {
+			return true;
```
Comment (rvdsteege) [inline adminer/index.php:21]: This poses a security issue, as this change removes the default passwordless login restriction. We only want to allow logging into SQLite databases.

```suggestion
			if ( defined( 'DRIVER' ) && 'sqlite' === DRIVER ) {
				return true;
			}

			return parent::login( $login, $password );
```

**Review by rvdsteege — CHANGES_REQUESTED** [review]

**Comment by rvdsteege** [comment]
I also noticed a fatal error on the 'Scanner' page:

```
Fatal error: Uncaught Error: Undefined constant "GLOB_BRACE" in /var/www/html/wp-content/plugins/wp-pronamic-client/admin/virus-scanner.php:78

Stack trace:
#0 /var/www/html/wp-content/plugins/wp-pronamic-client/classes/Plugin.php(135): include()
#1 /var/www/html/wp-content/plugins/wp-pronamic-client/classes/Admin.php(378): Pronamic\WordPress\PronamicClient\Plugin->display('admin/virus-sca…')
#2 /var/www/html/wp-includes/class-wp-hook.php(324): Pronamic\WordPress\PronamicClient\Admin->page_virus_scanner('')
#3 /var/www/html/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#4 /var/www/html/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /var/www/html/wp-admin/admin.php(259): do_action('pronamic_page_p…')
#6 {main} thrown in /var/www/html/wp-content/plugins/wp-pronamic-client/admin/virus-scanner.php on line 78
```

From the docs (https://www.php.net/manual/en/function.glob.php):

> Note: `GLOB_BRACE` is not available on some non GNU systems, like Solaris or Alpine Linux.

File: adminer/index.php (hunk 1 above)
Comment (remcotolsma) [reply adminer/index.php]: I had already considered this, but left it out for simplicity. Added for now, but doesn't necessarily eliminate the security issue.

Comment (rvdsteege) [reply adminer/index.php]: > but doesn't necessarily eliminate the security issue.

It prevents passworldless login to e.g. a MySQL server; as is the case by default. If you're running a SQLite database in production, then the issue persists indeed.

**Pull request merged**